spec/gorouter_templates_spec.rb

```diff
@@ -1509,7 +1509,7 @@
       { 'router' => {
         'port' => 81,
         'status' => { 'port' => 8081, 'tls' => {'port' => 8443}, },
-        'prometheus' => { 'port' => 7070 },
+        'prometheus' => { 'port' => 7777 },
```

Comment:
Let's hope this was just used in tests and no-one has this configured on a prod landscape. We should probably add a note to the release notes.

Reply:
true. it's a freely configurable value so we should be good. I'll add it to the release notes of the next release